examples/Factory_method/factory_method_example_1.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+from abc import ABCMeta, abstractmethod
+
+
+class GameClass(metaclass=ABCMeta):
+    def __init__(self):
+        self.hp = 10
+        self.mp = 10
+
+    @abstractmethod
+    def level_up(self):
+        pass
+
+    def print_status(self):
+        print('status hp: {} mp: {}'.format(self.hp, self.mp))
+
+
+class Warrior(GameClass):
+    def __init__(self):
+        super(Warrior, self).__init__()
+        self.hp += 10
+
+    def level_up(self):
+        self.hp += 15
+        self.mp += 5
+
+    def print_status(self):
+        print('Warrior ', end='')
+        super(Warrior, self).print_status()
+
+
+class Mage(GameClass):
+    def __init__(self):
+        super(Mage, self).__init__()
+        self.mp += 10
+
+    def level_up(self):
+        self.hp += 5
+        self.mp += 15
+
+    def print_status(self):
+        print('Mage ', end='')
+        super(Mage, self).print_status()
+
+
+class GameClassFactory(metaclass=ABCMeta):
+    @abstractmethod
+    def create_class(self):
```

Comment:
typing이 표현되면 좋을 것 같아요 
```suggestion
    def create_class(self) -> GameClass:
```